docs/api/throttling.md

```diff
@@ -0,0 +1,33 @@
+# NuGet API rate limiting
+
+I order to prevent the extreme overload on NuGet Server resources the NuGet.org service API needs some configurable and enforceable upper limits on acceptable API usage. 
+
+Initial planned throttling limits are based on the maximum load we've seen on these endpoints to date. Upon reaching the maximum allowed requests an error response returns. 
+
+    {
+      "statusCode": 429,
+      "message": "Rate limit is exceeded. Try again in 56 seconds."
+    }
+
+
```

Comment:
I'd put something here like:

The follow tables list the rate limits for the NuGet.org API.

Reply:
Updated.